numpyro/distributions/continuous.py

```diff
@@ -1082,3 +1082,34 @@ def mean(self):
     @property
     def variance(self):
         return (self.high - self.low) ** 2 / 12.
+
+
+@copy_docs_from(Distribution)
+class Logistic(Distribution):
+    arg_constraints = {'loc': constraints.real, 'scale': constraints.positive}
+    support = constraints.real
+    reparametrized_params = ['loc', 'real']
+
+    def __init__(self, loc=0., scale=1., validate_args=None):
+        self.loc, self.scale = promote_shapes(loc, scale)
+        batch_shape = lax.broadcast_shapes(np.shape(loc), np.shape(scale))
+        super(Logistic, self).__init__(batch_shape, validate_args=validate_args)
+
+    def sample(self, key, sample_shape=()):
+        z = random.logistic(key, shape=sample_shape + self.batch_shape + self.event_shape)
+        return self.loc + z * self.scale
+
+    @validate_sample
+    def log_prob(self, value):
+        log_exponent = (self.loc - value) / self.scale
+        log_denominator = np.log(self.scale) + 2 * np.log(1 + np.exp(log_exponent))
```

Comment:
nit: it is better to use `jax.nn.softplus(log_exponent)` instead of `np.log(1 + np.exp(log_exponent))`.

Reply:
Thanks for the tip @fehiepsi ! Just pushed this change.